sample/Kmd.Logic.FileSecurity.Client.ConfigurationSample/Program.cs

```diff
@@ -58,23 +60,63 @@ private static async Task Run(AppConfiguration configuration)
+
+            Log.Information("Fetching certificate details for certificate id {CertificateId} ", configuration.CertificateDetails.CertificateId);
+            var result = await fileSecurityClient.GetCertificate(certificateId).ConfigureAwait(false);
 
-                Log.Information("Fetching certificate details for certificate id {CertificateId} ", configuration.CertificateDetails.CertificateId);
-                var result = await fileSecurityClient.GetCertificate(certificateId).ConfigureAwait(false);
+            if (result == null)
+            {
+                Log.Error("Invalid certificate id {Id}", configuration.CertificateDetails.CertificateId);
+                return;
+            }
 
-                if (result == null)
-                {
-                    Log.Error("Invalid certificate id {Id}", configuration.CertificateDetails.CertificateId);
-                    return;
-                }
+            Console.WriteLine("Certificate ID: {0} \nCertificate Name: {1}\nSubscription ID : {2}", result.CertificateId, result.Name, result.SubscriptionId);
 
-                Console.WriteLine("Certificate ID: {0} \nCertificate Name: {1}\nSubscription ID : {2}", result.CertificateId, result.Name, result.SubscriptionId);
+            // Create a Sign Connfiguration 
+            var signConfigurationRequest = BuildSignConfigurationRequest();
+            Log.Information("Creating signconfiguration...");
+            var signConfigurationResult = await fileSecurityClient.CreateSignConfigurationPdf(signConfigurationRequest).ConfigureAwait(false);
+            if (signConfigurationResult == null)
+            {
+                Log.Error("Couldn't create signconfiguration");
+                return;
             }
+
+            Console.WriteLine(
+                "Configuration created successfully. SignConfiguration ID: {0} \nSignConfiguration Name: {1}\nSubscription ID : {2}",
+                signConfigurationResult.Id,
+                signConfigurationResult.Name,
+                signConfigurationResult.SubscriptionId);
+        }
+
+        private static SignConfigurationPdfRequestDetails BuildSignConfigurationRequest()
+        {
+            var pdfPrivilege = new PdfPrivilegeModel(
+                copyAllowLevel: 1,
+                changeAllowLevel: 1,
+                allowAssembly: true,
+                allowScreenReaders: true,
+                allowFillIn: true,
+                allowModifyAnnotations: true,
+                allowCopy: true,
+                allowModifyContents: true,
+                allowDegradedPrinting: true,
+                allowPrint: true,
+                printAllowLevel: 1,
+                allowAll: true,
+                forbidAll: false);
+            return new SignConfigurationPdfRequestDetails(
+                  signConfigurationId: Guid.Empty,
+                  name: "TestSignConfiguration",
+                  ownerPassword: "TestPwd",
+                  certificateId: Guid.Empty,
```

Comment:
Shouldn't these values be assigned from the configuration.

Reply:
Certificate Id can be used from Configuration in case we want the sign configuration to be created with certificate id. But don't think for `name` and `ownerpassword` we need to get from configuration as it is normal string values and for sample purpose we can hardcode in the program. Do you see any particular use case if we need to read it from the configuration?